hcl2template/types.packer_config.go

```diff
@@ -37,6 +38,9 @@ type PackerConfig struct {
 	provisionersSchemas packer.ProvisionerStore
 
 	postProcessorsSchemas packer.PostProcessorStore
+
+	except []glob.Glob
```

Comment:
felt unsure if this is the best approach but definitely the easiest one.

Reply:
This LGTM ! Now that I look at this I had completely omitted the fact that the only/except options also match post-processors

I think this is a tad confusing and I wish there was a `--skip-post-processor` option instead 🤔 .
But that's another topic !